sites/main-site/src/content/events/2025/hackathon-march-2025/harvard-chan-school.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Hackathon - March 2025 (Harvard Chan School)'
+subtitle: 'Local node of the nf-core hackathon at Harvard Chan School'
+shortTitle: 'Harvard Chan School'
+type: 'hackathon'
+startDate: '2025-03-17'
+startTime: '09:00+00:00'
```

Comment:
Offset is wrong

Reply:
I have a feeling that the dates are also wrong? 🤔

Reply:
Fixed